DeveloperGame/DeveloperGame/Question.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ComputerGeekGame
+{
+    public class Question
+    {
+        protected string _question;
+
+        public Question(string question)
+        {
+            _question = question;
+        }
+
+        public virtual void PrintQuestion()
+        {
+            Console.WriteLine(_question);
+        }
+    }
+
+    public class TrueOrFalseQuestion : Question
+    {
+        private bool _correctAnswer;
+
+        public TrueOrFalseQuestion(string question, bool correctAnswer) : base(question)
+        {
+            _correctAnswer = correctAnswer;
+        }
+
+        public override void PrintQuestion()
+        {
+            base.PrintQuestion();
+            Console.Write("\n\tTrue or False: ");
+        }
+    }
+
+    public class MultipleChoiceQuestion : Question
+    {
+        public enum Answers { A, B, C, D }
+
+        private Answers _correctAnswer;
+        private List<string> _answerOptions = new List<string>();
+
+        public MultipleChoiceQuestion(string question, string answerOption1, string answerOption2, string answerOption3, string answerOption4, Answers correctAnswer) : base(question)
+        {
+            _answerOptions.Add(answerOption1);_answerOptions.Add(answerOption2);_answerOptions.Add(answerOption3);_answerOptions.Add(answerOption4);
```

Comment:
This is really hard to read if you put it all on one line.

However, it might be easier to new up the list here instead of on line 43, so you can pass in the arguments as it constructs it.

Ex: _answerOptions = new List() {answerOption1, answerOption2, answerOption3, answerOption4};

Reply:
Done.